api-report/purchases-js.api.md

```diff
@@ -173,13 +173,37 @@ export interface Price {
     readonly formattedPrice: string;
 }
 
+// @public
+export interface PricingPhase {
+    readonly cycleCount: number;
+    readonly periodDuration: string | null;
+    readonly price: Price | null;
+}
+
 // @public
 export interface Product {
     readonly currentPrice: Price;
+    readonly defaultSubscriptionOption: SubscriptionOption | null;
     readonly displayName: string;
     readonly identifier: string;
     readonly normalPeriodDuration: string | null;
     readonly presentedOfferingIdentifier: string;
+    readonly subscriptionOptions: {

```

Comment:
This is pretty much unneeded for now (the backend should be the one choosing the offer for the SDK), but adding this to make sure the system is flexible if needed to choose between different options from the SDK.